spec/01_tic_tac_toe_spec.rb

```diff
@@ -85,40 +103,22 @@
     end
 
     describe '#valid_move?' do
-      it 'returns true/false based on position' do
+      it 'returns true/false based on whether the position is already occupied' do
         game = TicTacToe.new
         board = [" ", " ", " ", " ", "X", " ", " ", " ", " "]
         game.instance_variable_set(:@board, board)
 
-        position = "1"
+        position = 0
         expect(game.valid_move?(position)).to be_truthy
```

Comment:
Done @aturkewi — on another note, it would be great to test that `#position_taken?` is called within `#valid_move?` to verify that students aren't simply duplicating their code from `#position_taken?` inside of `#valid_move?`